radix_sort.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: UTF-8 -*-
+
+
+def int_radix(mylist):
+    '''
+    Sorts a list of ints by looking at the 1's place,
+    the 10's, 100's, etc.
+    '''
+    if len(mylist) <= 1:
+        return mylist
+    for i in mylist:
+        if (type(i) != int) or (i < 0):
+            raise TypeError('Please input valid integers')
+    if len(mylist) > 0:
+        max_list = max(mylist)
+        tens = 1
+        num_buckets = []
+        for i in range(10):
+            num_buckets.append([])
+        while max_list >= tens:
+            for item in mylist:
+                bucket = (item % (tens * 10) / tens)
```

Comment:
watch out for the true division here.  In Python 3 this would return floats when you really don't want it to.  Use floored division instead to be safe across all versions of Python.